packages/eslint-config-godaddy-flow/package.json

```diff
@@ -29,9 +29,13 @@
     "eslint": ">=6.0.0"
   },
   "dependencies": {
-    "babel-eslint": "^10.0.3",
+    "@babel/core": "^7.16.5",
```

Comment:
Could this be a peer + dev dependency instead? Duplicated `@babel/core` packages can be a problem. If moving to `peerDependencies`, I'd also suggest making it less restrictive, like `^7.0.0` in case the consumer has to be stuck on an older `@babel/core` for tragic reasons.